integration/tests/cook/util.py

```diff
@@ -1252,18 +1260,20 @@ def default_pool(cook_url):
 def all_pools(cook_url):
     """Returns the list of all pools that exist"""
     resp = session.get(f'{cook_url}/pools')
-    return resp.json(), resp
+    disallow_pools_regex = os.getenv('COOK_TEST_DISALLOW_POOLS_REGEX')
+    all_pools = resp.json() if disallow_pools_regex is None else \
+        [p for p in resp.json() if not re.match(disallow_pools_regex, p['name'])]
+    if disallow_pools_regex is not None:
+        for ii in resp.json():
+            logger.info(f"For Pool {ii['name']} we have {re.match(disallow_pools_regex, ii['name'])}")
```

Comment:
I would vote to either remove this log or make it a little more clear, e.g.:

```suggestion
            logger.info(f"For pool {ii['name']}, matching against the disallowed pools regex returns {re.match(disallow_pools_regex, ii['name'])}")
```